### PR TITLE
Provide with option to show language

### PR DESCRIPTION
**Pull request by qinezh**
@chenkennt @vwxyzh @superyyrrzz @hellosnow @ansyral

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs [hunk 1]
```diff
@@ -154,6 +155,15 @@ public static ApiBuildOutput FromModel(PageViewModel model)
                 return null;
             }
 
+            object displayLangs;
+            if (model.Metadata.TryGetValue("_displayLangs", out displayLangs))
+            {
+                var langs = ((IEnumerable)displayLangs).Cast<object>()
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs]: object[]

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs [hunk 2]
```diff
@@ -238,5 +248,15 @@ private static ApiBuildOutput FromModel(ItemViewModel model, Dictionary<string,
         {
             return crefs?.Select(c => ApiCrefInfoBuildOutput.FromModel(c, references, supportedLanguages)).ToList();
         }
+
+        private static string[] IntersectLangs(string[] defaultLangs, string[] displayLangs)
+        {
+            if (displayLangs != null && displayLangs.Length == 0)
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs]: from caller, displayLangs cannot be null.

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs [hunk 3]
```diff
@@ -154,6 +155,15 @@ public static ApiBuildOutput FromModel(PageViewModel model)
                 return null;
             }
 
+            object displayLangs;
+            if (model.Metadata.TryGetValue("_displayLangs", out displayLangs))
+            {
+                var langs = ((IEnumerable)displayLangs).Cast<object>()
+                                 .Select(x => x.ToString())
+                                 .ToArray();
+                model.Items[0].SupportedLanguages = IntersectLangs(model.Items[0].SupportedLanguages, langs);
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs]: also update other items, like: model.Items.Foreach(i => i.SupportedLanguages = ...)

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**